src/components/mermaidMap/ProjectSitesMap/ProjectSitesMap.js

```diff
@@ -43,6 +43,7 @@ const ProjectSitesMap = ({ sitesForMapMarkers, choices }) => {
       customAttribution: language.map.attribution,
     })
 
+    // adds zoom and rotation controls to the map.
     addMapController(map.current)
```

Comment:
I prefer changing function names if possible to adding comments in general

Reply:
will rename to `addZoomController`